doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/configuring-autoscaling.rst

```diff
@@ -2,55 +2,44 @@
 
 .. _deployment-guide-autoscaler-under-construction:
 
-Autoscaling with Ray
---------------------
+Configuring the Ray Autoscaler
+==============================
 
-Ray is designed to support highly elastic workloads which are most efficient on
-an autoscaling cluster. At a high level, the autoscaler attempts to
-launch/terminate nodes in order to ensure that workloads have sufficient
-resources to run, while minimizing the idle resources.
+This page provides an overview of the parameters for the Ray autoscaler. A Ray cluster runs an autoscaler that can modify
+the number of nodes based on the resources set by task, actor or placement group. The autoscaler only considers logical resources (e.g. those specified
+in ray.remote and displayed in ray status), with the exception of object_store_memory which it does not consider. If a user tries to launch
```

Comment:
Eh...the `with` part is a little confusing. Maybe we can drop the mention of `object_store_memory` here and just mention it when explaining how downscaling works.